### PR TITLE
Blog post Q3 2023

### PR DESCRIPTION
**Pull request by choldgraf**
This is a brief update for Q3 2023. It's a bit later than intended but I wanted to get all the near-term job postings up before getting this out.

It describes some of the stuff our team has identified and been working through over the past few months, and describes some of the major organizational shifts we're looking to make to adjust.

### CODE REVIEW

File: content/blog/2023/2023-q3-update/index.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+If you think you'd be interested in this role, please consider applying!
+
+### We need a more structured system of work and execution
+
+Next, we realized that there are many different workstreams we must balance simultaneously.
+There are new communities to be onboarded, infrastructure bugs to be fixed, questions to be answered, and strategic priorities to improve the service.
+Expecting each team member to individually find the right balance between all of these is increasingly unrealistic.
+We need capacity to oversee this system of work and bring some structure to it.
+
+So, we're creating a **Deliver Manager role** to oversee the system of work that our engineering team uses for its planning and execution.
```
Comment (yuvipanda) [inline content/blog/2023/2023-q3-update/index.md]:
```suggestion
So, we're creating a **Delivery Manager role** to oversee the system of work that our engineering team uses for its planning and execution.
```

File: content/blog/2023/2023-q3-update/index.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+We are just onboarding our first communities and hope to grow that service in the coming year.
+
+Finally, we've been fortunate to receive some grants around creating content and designing workflows that utilize cloud infrastructure. For example, a [NASA TOPS grant](https://nasa.github.io/Transform-to-Open-Science/open-science-101/) and an upcoming collaboration with [Project Pythia](https://projectpythia.org/) around geospatial workflows.
+
+### Financial picture
+
+At this point, we're **recovering roughly 40% of our operating costs** through recurring fees of our managed hub service (making up the remainder in development contracts and grants), and **we've currently got around 2 years of runway**.
+
+Both of these will be lowered soon, however, because we've about to hire for several more positions.
+Improvements to our product model will allow us to estimate and recovery our service costs more effectively, but we also intend to raise some funds next year to support our efforts in making our service more robust, sustainable, and valuable to communities.
```
Comment (mfisher87) [inline content/blog/2023/2023-q3-update/index.md]:
```suggestion
Improvements to our product model will allow us to estimate and recover our service costs more effectively, but we also intend to raise some funds next year to support our efforts in making our service more robust, sustainable, and valuable to communities.
```

File: content/blog/2023/2023-q3-update/index.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+This became clear in our first in-person team meeting last May.
+At that meeting, we realized that many on the team were spending too much of their time "reacting" to demands from the service.
+We also learned that the scope and complexity of our various workstreams had gotten to a size where our informal team structures of work prioritization were no longer adequate.
+
+## Growing the complexity of our team to match the complexity of our service
+
+So, for the past several months we've been working on a plan to evolve 2i2c's team structure in order to more effectively manage the complexity of our service, and balance long- and short-term thinking.
+
+This began with an organizational audit carried out by [makeadifference.digital](https://www.makeadifference.digital/), a consulting group that focusese on tech-for-good and non-profit products and services.
```
Comment (mfisher87) [inline content/blog/2023/2023-q3-update/index.md]:
```suggestion
This began with an organizational audit carried out by [makeadifference.digital](https://www.makeadifference.digital/), a consulting group that focuses on tech-for-good and non-profit products and services.
```

**Review by mfisher87 — COMMENTED** [review]
Hope you don't mind me sharing typos I found while following along ;)

File: content/blog/2023/2023-q3-update/index.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+### We need a more structured system of work and execution
+
+Next, we realized that there are many different workstreams we must balance simultaneously.
+There are new communities to be onboarded, infrastructure bugs to be fixed, questions to be answered, and strategic priorities to improve the service.
+Expecting each team member to individually find the right balance between all of these is increasingly unrealistic.
+We need capacity to oversee this system of work and bring some structure to it.
+
+So, we're creating a **Deliver Manager role** to oversee the system of work that our engineering team uses for its planning and execution.
+In addition, this role will serve at an organization-wide capacity as an **interrim Chief of Staff**.
```
Comment (colliand) [inline content/blog/2023/2023-q3-update/index.md:86]: In addition, this role will serve at an organization-wide capacity as an **interim Chief of Staff**.

Comment (colliand) [inline content/blog/2023/2023-q3-update/index.md]:
```suggestion
In addition, this role will serve at an organization-wide capacity as an **interim Chief of Staff**.
```

**Pull request merged**